docs/src/guide/installation.md

```diff
@@ -30,6 +30,40 @@ additional build system documentation.
 Please help us [keep these instructions up-to-date][contribute]!
 
 
+## Linux
+
+For Ubuntu 20.04LTS, these commands install the basic build system:
+
+	sudo apt-get install python3 python3-pip scons git gcc-arm-none-eabi openocd \
+	  gcc build-essential libboost-all-dev
+	pip3 install modm gdbgui
+
+!!! warning "OpenOCD for recent targets"
+	The latest OpenOCD release v0.10.0 (as of May 2020) is too old for some targets
+	(STM32G0, STM32G4, STM32F7). To program these targets you need to compile the
+	[HEAD version of OpenOCD from source][openocd-source].
```

Comment:
Same issue here, installing first the old openocd, then adding the new openocd to the path.

Reply:
If the user gets to this warning by reading it, I think he is competent to remove the OpenOCD package from the apt command.

Reply:
_... compile [OpenOCD], install it and add it to the beginning of your `$PATH`_.
In case the user installed openocd with the command above or openocd was previously installed the up-to-date OpenOCD will be used.